### ISSUE TITLE
Question: how to mute channels

### ISSUE THREAD

**Issue by matthiasnoback** [symfony/monolog-bundle#102]
In order to "mute" log messages from the "event" channel, I have this configuration in my bundle:

```yaml
monolog:
    handlers:
        main:
            type:  stream
            path:  %kernel.logs_dir%/%kernel.environment%.log
            level: debug
            channels: [""!event"]
        mute:
            type: "null"
            channels: ["event"]
```

This is effective, but are there other ways to mute channels? What happens by default with messages that are excluded by any of the handlers? I noticed them popping up in the output of the PHPStorm PHPUnit test runner, but I couldn't find any piece of code in the bundle or the library that could be the cause of this.

**Comment by fabpot** [comment]
fixed by #177 (I would have loved to read this before investigating the issue, it would have saved me some time :))

**Comment by Seldaek** [comment]
Thanks @fabpot, for the record monolog 2.0 doesn't do this default handler thing anymore, but it's not yet stable either.
